server/sonar-db-dao/src/test/java/org/sonar/db/qualityprofile/QualityProfileDaoTest.java

```diff
@@ -140,11 +140,11 @@ public void find_all_is_sorted_by_profile_name() {
   public void get_default_profile() {
     dbTester.prepareDbUnit(getClass(), "shared.xml");
 
-    QualityProfileDto java = underTest.selectDefaultProfile(dbTester.getSession(), "java");
+    QualityProfileDto java = underTest.selectDefaultProfile(dbTester.getSession(), organization, "java");
```

Comment:
the test should be improved to verify that condition on organization is applied. Currently the test passes if I drop `AND p.organization_uuid = #{organizationUuid, jdbcType=VARCHAR}` from SQL request.